wolfssl/wolfcrypt/visibility.h

```diff
@@ -33,47 +33,65 @@
     (!defined(HAVE_FIPS_VERSION) || (HAVE_FIPS_VERSION < 2))
     #include <cyassl/ctaocrypt/visibility.h>
     #define WOLFSSL_API   CYASSL_API
+    #define WOLFSSL_ABI   CYASSL_API
```

Comment:
Why complicate this to make yet another define that has to be maintained in several places perfectly?  What was wrong the ABI meaning nothing except to us?

Reply:
For clarification, do you mean just this line, or having WOLFSSL_ABI have a visibility rule?